src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+    private double hypotenuse;
+
+    public RightTriangle(String color,
+                         double firstLeg,
+                         double secondLeg,
+                         double hypotenuse) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+        this.hypotenuse = hypotenuse;
```

Comment:
The hypotenuse should not be passed as a parameter to the constructor. It is a derived property that can be calculated from the firstLeg and secondLeg. Remove this parameter and calculate the hypotenuse within the class if needed.